app/models/container_build.rb

```diff
@@ -0,0 +1,14 @@
+class ContainerBuild < ActiveRecord::Base
+  include CustomAttributeMixin
+  include ReportableMixin
```

Comment:
I'm not sure we need CustomAttributeMixin, ReportableMixin, has_many :lables and acts_as_miq_taggable.
we can take a look on Sunday. maybe @zeari knows